File: Semana6_PY/ejer1.py
num=int(input("Ingrese número de la tabla: "))

while num<=0:
    num=int(input("Número inválido: Ingrese número de la tabla: "))



i=1
print()
while i<=12:
    print(f"{num}x{i}={num*i}")
    i+=1

File: SEMANA11_YORDAN/ejer1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SEMANA11_YORDAN
{
    internal class ejer1
    {
        static void Main(string[] args)
        {
            int[] num = new int[5];
            for(int i=0; i<num.Length; i++)
            {
                Console.Write($"Ingrese número {i + 1}: ");
                num[i] = int.Parse(Console.ReadLine());

            }
            Console.WriteLine("\nNúmeros ingresados: ");
            foreach(int i in num)
            {
                Console.WriteLine(i);
            }
            Console.ReadKey();
        }
    }
}
